examples/hello_world.rs

```diff
@@ -42,11 +42,11 @@ impl RootContext for HelloWorld {
     fn on_tick(&mut self) {
         cfg_if! {
             if #[cfg(all(target_arch = "wasm32", target_os = "unknown"))] {
-                let now: DateTime<Utc> = self.get_current_time().into();
+                let now: OffsetDateTime = OffsetDateTime::now_utc();
```

Comment:
This isn't equivalent replacement, and most likely doesn't work at all, since there is no `std::time::SystemTime` for the `wasm32-unknown-unknown` platform.

I think this should be `OffsetDateTime::from_unix_timestamp_nanos(self.get_current_time());`

Reply:
Good catch, hadn't consider that. Updated to use `get_current_time`. Verified local with with `wasm32-unknown-unknown` build target.

```
example_envoy_1 | [2022-01-31 13:35:41.634][1][debug][upstream] [source/common/upstream/strict_dns_cluster.cc:178] DNS refresh rate reset for echo_service, refresh rate 5000 ms
example_envoy_1 | [2022-01-31 13:35:45.770][19][info][wasm] [source/extensions/common/wasm/context.cc:1172] wasm log rbi_filter_id rbi_injection_vm_id: It's 2022-01-31 13:35:45.770877 +00:00:00, there is no lucky number.
```